zcash_client_backend/src/data_api/testing/pool.rs

```diff
@@ -4954,3 +4957,114 @@ pub fn receive_two_notes_with_same_value<T: ShieldedPoolTester>(
         assert_eq!(T::note_value(note.note()), value);
     }
 }
+
+fn build_coinbase_tx(
+    network: &LocalNetwork,
+    value: Zatoshis,
+    recipient: TransparentAddress,
+    miner_data: Option<PushValue>,
+) -> zcash_primitives::transaction::builder::BuildResult {
+    let build_config = BuildConfig::Coinbase { miner_data };
+    let mut builder = Builder::new(network.clone(), BlockHeight::from_u32(1), build_config);
```

Comment:
For fidelity, this shouldn't hardcode the target height, but should take it as an argument.